src/twitter/Timeline.js

```diff
@@ -97,40 +132,46 @@ class Timeline {
       obj = this._hashtags[htag]
     }
     else {
-      obj = {hashtag, room: [] }
+      obj = {hashtag, room: new Set() }
       if(opts.is_new) {
         this._newtags.add("#"+hashtag);
       }
     }
-    if(!obj.room.includes(room_id)) {
-      obj.room.push(room_id);
-    }
+    obj.room.add(room_id);
     if(htag !== -1) {
-      obj = this._hashtags[htag] = obj;
+      this._hashtags[htag] = obj;
     }
     else {
       this._hashtags.push(obj);
     }
     log.info("Added Hashtag: %s", hashtag);
+    return true;
   }
 
   /**
    * Add a Twitters user's timeline to the timeline processor. Tweets will be
    * automatically send to the given room.
    *
+   * HTM* - Use a single account to send tweets, avoiding large numbers of join
+   * events. This is enabled automatically unless disabled via config
```

Comment:
This looks to be mistakenly old code that I probably wanted to use before I changed the way high traffic mode worked. It's now done upon receiving new tweets so this doesn't need to exist at all.